about.html

```diff
@@ -105,7 +105,7 @@
   - name: Rollin Thomas
     photo: Rollin_Thomas.jpg
     linkedin: https://www.linkedin.com/in/rollin-thomas-507185b5/
-    personal: https://www.nersc.gov/about/nersc-staff/programming-environments-and-models/rollin-thomas/
+    personal: https://profiles.lbl.gov/20396-rollin-thomas
```

Comment:
I added this because the NERSC page is now broken due to a site move, and this seems to be the updated staff page for Rollin. This was causing a test failure